packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -63,8 +63,13 @@ Future<T> debugInstrumentAction<T>(String description, Future<T> action()) {
   }
 }
 
-/// Arguments to whitelist [Timeline] events in order to be shown in the
-/// developer centric version of the Observatory Timeline.
+/// Argument passed to [Timeline] events in order to cause those events to be
+/// shown in the developer-centric version of the Observatory Timeline.
+///
+/// See also:
+///
+///  * [Timeline.startSync], which typically takes this value as its `arguments`
```

Comment:
nit that cannot be fixed in this PR: I wish [Timeline.startSync] would explain what the `arguments` argument is used for. Docs just state "Optionally takes a Map of arguments." - Thank you, I can see that from the method signature...

Reply:
file a bug on the dart sdk?